packages/algorand-js/src/types.ts

```diff
@@ -121,3 +121,14 @@ export enum EncodingType {
   HEX,
   UTF8
 }
+
+export interface AccountsMap {
+  [addr: string]: AccountState
+}
+
+export interface Storage {
+  accounts: Map<string, AccountState>
+  globalApps: Map<number, SSCParams>
+  tx: Txn // current txn
+  gtxs: Txn[] // all transactions
```

Comment:
`tx` and `gtxs` is not a part of State. This is a part of the current transaction context / runtime context.

Reply:
Okay, sorry. I am adding this as a part of `runtime.ctx`